README.md

```diff
@@ -23,14 +23,25 @@ As a benefit, this implementation is cross-platform (x64 linux/win are tested, p
 
 ### Changelog
 
+#### Changes in version 3.3.Y
+
+*Breaking Change / Warning*: This change removes the key of IgnoreFolders and IgnoreFiles per 3.2.Y and in favour of an "Ignore" key with support for [File Globbing support](https://docs.microsoft.com/en-us/dotnet/core/extensions/file-globbing).
+Simply add the following to CompilerSettings.json:
+
+```json
+  "CompilerSettings": {
+     "Ignore": [ "wwwroot/*.scss", "wwwroot/css/specific-file.scss", "wwwroot/_lib/**/*.scss", "bin/**/*", "obj/**/*" ],
+  }
+```
+
 #### Changes in version 3.2.Y
 
-Supports excluding certain files and folders when using recursive mode
+Supports excluding certain files and folders (nb. Sub folders must be specified specifically if wanting to be ignored) when using recursive mode
```

Comment:
This wouldn't be the case with v3.3.0, right? My expectation for ignoring a folder is that the tool never visits that folder, i.e. including all subfolders.

Reply:
Yes v3.3 replaces the distinction between IgnoreFiles and IgnoreFolders in favour of an Ignore key with File Globbing.
I've left these notes in 3.2 as 3.2 still exists (albeit being a preview version we could remove these notes maybe?)

Reply:
I think the notes should be kept. Doesn't hurt in any case.